index.js

```diff
@@ -56,6 +57,20 @@ var argv = yargs.scriptName('cypress-utils')
     default: 'cypress/integration',
     global: true,
   })
+  .option('fileFormat', {
+    alias: ['f', 'fileFormat'],
+    type: 'string',
+    description: 'Expected file format',
+    default: '.spec.js',
+    global: true,
+  })
+  .option('testFiles', {
+    alias: ['testFiles'],
+    type: 'string',
+    description: 'A String or Array of glob patterns of the test files to load',
+    default: '**/*.*',
+    global: true,
+  })
```

Comment:
I feel like these should be a single property. It makes sense to use the Cypress configuration's `testFiles` property, which itself should hold enough info to supplement for `fileFormat`.

Reply:
Yep I agree that would work 👍